docs/rules/comma-spacing.md

```diff
@@ -13,25 +13,26 @@ var foo = 1 ,bar = 2;
 
 This rule aims to enforce spacing around a comma. As such, it warns whenever it sees a missing or unwanted space in commas of variable declaration, object property, function parameter, sequence and array element.
 
+This rule does not apply in an `ArrayExpression` or `ArrayPattern` to either of the following:
 
-## Options
-
-The rule takes one option, an object, which has two keys `before` and `after` having boolean values `true` or `false`. If `before` is `true`, space is enforced before commas and if it's `false`, space is disallowed before commas. If `after` is `true`, space is enforced after commas and if it's `false`, space is disallowed after commas. The default is `{"before": false, "after": true}`.
+* adjacent null elements
+* an initial null element, to avoid conflicts with the [`array-bracket-spacing`](array-bracket-spacing.md) rule
 
-```json
-    "comma-spacing": ["error", {"before": false, "after": true}]
-```
+## Options
 
-The following examples show two primary usages of this option.
+This rule has an object option:
 
-### `{"before": false, "after": true}`
+* `"before": false` (default) disallows space before commas
```

Comment:
"space" => "spaces" (to be consistent with previous cases where "spaces" is used) ?